common/src/main/kotlin/orion/data/Task.kt

```diff
@@ -1 +1,24 @@
-package orion.data
\ No newline at end of file
+package orion.data
+import kotlinx.serialization.*
+
+// TODO - get this working 
+// 1 (highest) to 4 (lowest)
+// @Serializable
+// enum class Priority(val rgb: String)  { 
+//     P1("1"),
+//     P2("2"),
+//     P3("3"),
+//     P4("4") 
+// }
+
+// TODO - recurrence + subtasks 
+@Serializable
+data class Task(
+    val Id: String = "",
+    val CreatedAt: String = "",
+    var Description: String = "",
+    var Deadline: String = "",
+    var Priority: Int = 1,
```

Comment:
I think `Priority` should probably be an `Int?`, and have 1, 2, 3 as the different priority levels. You can opt out of applying priority at all though, in which case it's null. I would also be open to have it be an `enum?` with `LOW`, `MEDIUM`, and `HIGH` values. Might be cleaner what which end of the scale is (a la Matt's confusion in our meeting)